Documentation/content/examples/index.md

```diff
@@ -201,16 +382,25 @@ This will allow you to see the some live code running in your browser. Just pick
 [ShapeWidget]: ../docs/gallery/ShapeWidget.png
 [SphereWidget]: ../docs/gallery/SphereWidget.jpg
 [SplineWidget]: ../docs/gallery/SplineWidget.gif
+[Box]: ../docs/gallery/Box.jpg
 
 ## Connectivity
 
 <div class="gallery">
 
+[![AR Example][AR]](./AR.html "AR with WebXR")
+[![LookingGlass Example][LookingGlass]](./LookingGlass.html "Render scene into a LookingGlass device")
+[![VR Example][VR]](./VR.html "VR with WebXR")
+[![ItkWasmVolume Example][ItkWasmVolume]](./ItkWasmVolume.html "ItkWasmVolume")
 [![RemoteView Example][RemoteViewWithLogos]](./RemoteView.html "Connect a VTK or ParaView Python backend server via WebSockets")
 [![ImageStream Example][ImageStreamWithLogos]](./ImageStream.html "Stream a ParaView Python backend server via WebSockets under a VTK.js rendering")
 
 
 </div>
 
+[AR]: ../docs/gallery/ArCone.jpg
+[LookingGlass]: ../docs/gallery/LookingGlassCone.png

```

Comment:
please add a LookingGlass logo onto the screenshot (and save it as `...WithLogo.jpg`)